docs/data/pages.ts

```diff
@@ -165,6 +165,7 @@ const pages: MuiPage[] = [
               },
             ],
           },
+          { pathname: '/x/react-data-grid/ask-your-table', plan: 'premium', unstable: true },
```

Comment:
```suggestion
          { pathname: '/x/react-data-grid/ai-assistant', plan: 'premium', unstable: true },
```

@joserodolfofreitas the page should be `ai-assistant` rather than `ask-your-table`, right?

Reply:
> And AI Assistant in the docs sidebar?

Right. **AI Assistant** doesn't fully describe the agents "function", hence we're complement titles with "Ask You Table". However:
1. it's still a more generic term people will transfer from other products, and therefore, likely to be more searched and widely understood.
2. On the actual UI panel, a less branded title definitely blends better with customers' applications.